fia_auth/model.py

```diff
@@ -31,6 +31,7 @@ class User:
     """Internal User Model for packing JWTs"""
 
     user_number: int
+    users_name: str
```

Comment:
This is minor but can we make this user_name, since the token stores it as `username` anyway